stl/inc/tuple

```diff
@@ -209,18 +210,18 @@ public:
     constexpr tuple(const tuple&) noexcept /* strengthened */ {} // TRANSITION, ABI: should be defaulted
 
     template <class _Alloc>
-    tuple(allocator_arg_t, const _Alloc&) noexcept /* strengthened */ {}
+    _CONSTEXPR20 tuple(allocator_arg_t, const _Alloc&) noexcept /* strengthened */ {}
 
     template <class _Alloc>
-    tuple(allocator_arg_t, const _Alloc&, const tuple&) noexcept /* strengthened */ {}
+    _CONSTEXPR20 tuple(allocator_arg_t, const _Alloc&, const tuple&) noexcept /* strengthened */ {}
 
     template <class _Tag, enable_if_t<is_same_v<_Tag, _Exact_args_t>, int> = 0>
     constexpr tuple(_Tag) noexcept /* strengthened */ {}
 
     template <class _Tag, class _Alloc, enable_if_t<is_same_v<_Tag, _Alloc_exact_args_t>, int> = 0>
-    tuple(_Tag, const _Alloc&) noexcept /* strengthened */ {}
+    _CONSTEXPR20 tuple(_Tag, const _Alloc&) noexcept /* strengthened */ {}

```

Comment:
Ditto "plain `constexpr` because internal machinery". (And on 255, 261, and 264.)

Reply:
Done